src/pages/events/2020/03/sugcon.md

```diff
@@ -0,0 +1,22 @@
+---
+templateKey: 'event-page'
+eventId: 
+title: SUGCON - 26 & 27 March 2020
+sup: The Sitecore User Group Conference (SUGCON) brings the Sitecore user community together and provides the perfect platform to gain knowledge and get inspired by the potential within Sitecore Experience Cloud, Experience Commerce, Content Hub and more. After many very successful events, we are expecting over 500 attendees to attend the event..
+date: 2020-03-26
+dateConfirmed: false

```

Comment:
Can be true

Reply:
Had it to false as it's 2 days (26 + 27). Now set to start date/time of session